pkg/alertmanager/operator.go

```diff
@@ -1624,6 +1624,10 @@ func checkMSTeamsConfigs(
 			return err
 		}
 
+		if config.Summary != nil && amVersion.LT(semver.MustParse("0.27.0")) {
+			return fmt.Errorf(`Summary only available in Alertmanager >= 0.27.0`)
+		}
```

Comment:
move to `func (tc *msTeamsConfig) sanitize(amVersion semver.Version, logger log.Logger) error {...}`

Reply:
Hi @dongjiang1989 changed accordingly, could you review?